### PR TITLE
Order test

### PR DESCRIPTION
**Pull request by ewelkaw**
Test for single active basket

### CODE REVIEW

File: ewardrobe/ewardrobe_app/queries/order.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ):
+        self.__product_id = product_id
+        self.__user = user
+        self.__amount = amount
+        self.__size = size
+
+    def add_product_to_basket(self) -> dict:
+        product = Product.objects.get(id=self.__product_id)
+        basket, _ = Basket.objects.get_or_create(status=0, user=self.__user)
+        basket.save()
```
Comment (esse) [inline ewardrobe/ewardrobe_app/queries/order.py]: do we need to use `save()` here?

File: ewardrobe/ewardrobe_app/queries/order.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        product_amount, _ = ProductsAmount.objects.get_or_create(
+            basket=basket, product=product, size=self.__size
+        )
+        product_amount.amount += int(self.__amount)
+        product_amount.save()
+
+        products_amounts = ProductsAmount.objects.filter(basket=basket).all()
+        total_cost = sum(products_amount.cost for products_amount in products_amounts)
```
Comment (esse) [inline ewardrobe/ewardrobe_app/queries/order.py]: it should be done using sql

File: ewardrobe/ewardrobe_app/tests/test_order_query.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = User.objects.create_user(
+            username="john", email="lennon@thebeatles.com", password="johnpassword"
+        )
+        cls.brand = Brand.objects.create(name="brand")
+        cls.category = Category.objects.create(name="category")
+        cls.retailer = Retailer.objects.create(name="retailer")
+        cls.color = Color.objects.create(name="color")
+        cls.product1 = Product.objects.create(
```
Comment (esse) [inline ewardrobe/ewardrobe_app/tests/test_order_query.py]: maybe it should be in factory?

File: ewardrobe/ewardrobe_app/queries/order.py (hunk 1 above)
Comment (ewelkaw) [reply ewardrobe/ewardrobe_app/queries/order.py]: yup :D

File: ewardrobe/ewardrobe_app/queries/order.py (hunk 2 above)
Comment (ewelkaw) [reply ewardrobe/ewardrobe_app/queries/order.py]: added comment

**Pull request merged**